addons/sys_attenuate/XEH_preInit.sqf

```diff
@@ -10,4 +8,6 @@ PREP_RECOMPILE_END;
 
 GVAR(attenuationCache) = HASH_CREATE;
 
-ADDON = true;
+if (!hasInterface) exitWith {
```

Comment:
One thing I meant when I said streamline preInit files is things like this. In this case it should look like this for easiest readability:
```
#include "script_component.hpp"

ADDON = false;

PREP_RECOMPILE_START;
#include "XEH_PREP.hpp"
PREP_RECOMPILE_END;

if (hasInterface) then {
    GVAR(attenuationCache) = HASH_CREATE;
};

ADDON = true;
```